tests/test_operation_definition.py

```diff
@@ -72,18 +81,6 @@ def test_get_definition():
 ################################################################################
 
 
-class InvalidTypedFieldTestOp(IRDLOperation):
-    name = "test.invalid_typed_field"
-
-    field: int
-
-
-def test_invalid_typed_field():
```

Comment:
This was a casualty of the transition. We no longer catch fields that aren't valid. I would say this is worth the sacrifice.

Reply:
Emilien made me aware of __annotations__, I'll see if I can resurrect this using string matching